pyproject.toml

```diff
@@ -101,20 +101,67 @@ commit_message = "Version generated by python-semantic-release [ci skip]"
 [tool.poetry.plugins."pytest11"]
 fastapi-sqla = "fastapi_sqla._pytest_plugin"
 
-[tool.pylama]
-paths = "fastapi_sqla tests"
-linters = "pycodestyle,mccabe,pyflakes"
+[tool.ruff]
+target-version = "py39"
+
+[tool.ruff.lint]
+select = [
+  "E", # pycodestyle
+  "W", # pycodestyle warnings
+  "F", # pyflakes
+  "I", # isort
+  "N", # pep8-naming
+  "D", # pydocstyle
+  "UP", # pyupgrade
+  "ASYNC", # flake8-async
+  "S", # flake8-bandit
+  "B", # flake8-bugbear
+  "C4", # flake8-comprehensions
+  "SIM", # flake8-simplify
+  "TD", # flake8-todos
+  "ERA", # eradicate
+  "PL", # pylint
+  "RUF" # ruff
+]
+
+# Ignore some pydocstyle rules that Google convention enables
+ignore = [
```

Comment:
Standard Dialogue ignores with the exception of `N802`, `N803` and `N806` given the pervasive use of title cased functions as FastAPI dependencies.